validation/MissionSequence.hpp

```diff
@@ -0,0 +1,25 @@
+
+#include <any>
+#include <iostream>
+#include <map>
+
+class MissionSequence
+{
+   public:
+    struct ComparisonResults
+    {
+        Position OSTkPosition_Minus_OtherPosition_RMS_;
+        Position OSTkVelocity_Minus_OtherVelocity_RMS_;
+    };
+
+    MissionSequence(const std::map<std::string, std::any>& data_tree);
```

Comment:
```suggestion
    MissionSequence(const std::map<std::string, std::any>& dataTree);
```

and adapt casing elswhere